incubator/hnc/internal/reconcilers/object.go

```diff
@@ -43,6 +43,7 @@ import (
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/forest"
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/metadata"
 	"sigs.k8s.io/multi-tenancy/incubator/hnc/internal/object"
+	selectorUtil "sigs.k8s.io/multi-tenancy/incubator/hnc/internal/pkg/selector"
```

Comment:
Nit: consider renaming the package to `selectors` (like the Go standard library `strings` or `errors`) and then import it without renaming it here.

We should probably rename the `object` package to `objects` as well but not in this PR. ("metadata" is already plural, and there's usually only one "forest" so singular makes sense there IMO).

Reply:
done